### PR TITLE
feat(#23): triggering fireworks when everyone's on board 🎆

### PR DESCRIPTION
**Pull request by OllyNicholass**

### CODE REVIEW

**Comment by jcreek** [comment]
The fireworks look great, but I have concerns about the sound as it's very loud and far too long. A simple 'tada' sound would be perfect for this moment, at a reduced volume and only very short as users have no way of muting this noise.

File: src/routes/estimation/[roomId]/+page.svelte [hunk 1]
```diff
@@ -89,6 +90,23 @@
 	let showEstimates = false;
 	let disableEstimates: boolean = false;
 	let audioElement;
+	let showFireworks = false;
+	let fireworks: Fireworks
+  let fireworkOptions: FireworksOptions = {
```
Comment (jcreek) [inline src/routes/estimation/[roomId]/+page.svelte:95]: Formatting needs fixing here

File: src/routes/estimation/[roomId]/+page.svelte [hunk 2]
```diff
@@ -244,17 +274,22 @@
 	<EstimateGroupsList {estimateGroups} />
 {/if}
 
+{#if showFireworks}
+	<div class="launch-pad">
```
Comment (jcreek) [inline src/routes/estimation/[roomId]/+page.svelte:283]: I'm not sure this class name is particularly descriptive, although it is a very strong image!

File: src/routes/estimation/[roomId]/+page.svelte [hunk 3]
```diff
@@ -344,12 +379,21 @@
...
+	}
 
 	#tooltip-container ul li {
-        text-indent: -10px;
-    }
+		text-indent: -10px;
+	}
+
+	.launch-pad {
+    position: fixed;
```
Comment (jcreek) [inline src/routes/estimation/[roomId]/+page.svelte:397]: Formatting

**Review by jcreek — CHANGES_REQUESTED** [review]

**Review by jcreek — APPROVED** [review]

**Comment by jcreek** [comment]
Thank you for your contribution @OllyNicholass :)

File: src/routes/estimation/[roomId]/+page.svelte (hunk 1 above)
Comment (OllyNicholass) [reply src/routes/estimation/[roomId]/+page.svelte]: Should now be fixed 👍

File: src/routes/estimation/[roomId]/+page.svelte (hunk 2 above)
Comment (OllyNicholass) [reply src/routes/estimation/[roomId]/+page.svelte]: 🚀

File: src/routes/estimation/[roomId]/+page.svelte (hunk 3 above)
Comment (OllyNicholass) [reply src/routes/estimation/[roomId]/+page.svelte]: Should now be fixed 👍

**Pull request merged**